.eleventy.js

```diff
@@ -62,20 +60,11 @@ module.exports = function (eleventyConfig) {
 
 	// only content in the `posts/` directory
 	eleventyConfig.addCollection("posts", function (collection) {
-		return collection.getFilteredByGlob("./src/posts/*").sort(function (a, b) {
+		return collection.getFilteredByGlob("./posts/*.md").sort(function (a, b) {
```

Comment:
5️⃣ these tweks were necessary to prevent a `PrematureUsage` error at build time